lib/ember-orbit/live-query.js

```diff
@@ -0,0 +1,41 @@
+import { operationType } from 'orbit-common/lib/operations';
+
+export default Ember.ArrayProxy.extend({
+  _orbitCache: null,
+  _query: null,
+  _identityMap: null,
```

Comment:
I had half an eye on lazy evaluation of live queries, it would be nice to be able to do AREL style `store.liveQuery(oql).where(oql)` for instance. This isn't implemented yet but all the parts are here.